contracts/staking/token/TokenPoolsManager.sol

```diff
@@ -0,0 +1,196 @@
+//_ SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import "@openzeppelin/contracts/proxy/beacon/UpgradeableBeacon.sol";
+import "@openzeppelin/contracts/proxy/beacon/BeaconProxy.sol";
+import "./TokenPool.sol";
+import "./LimitedTokenPool.sol";
+import "../../funds/RewardsBank.sol";
+import "../../LockKeeper.sol";
+
+contract TokenPoolsManager is AccessControl{
+    LockKeeper lockKeeper;
+    RewardsBank public bank;
+    UpgradeableBeacon public tokenPoolBeacon;
+    UpgradeableBeacon public depositedTokenPoolBeacon;
+
+    mapping(string => address) public pools;
+    mapping(string => address) public depositedPools;
```

Comment:
1) why use names and not addresses directly?
2) maybe two different managers for pools and depositedPools?